AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java

```diff
@@ -281,7 +281,16 @@ else if (ingestDocument.hasField("COMM")){
                 if (kubernetes.containsKey("namespace_name")) {
                     namespaceName = ((String)kubernetes.get("namespace_name"));
                 } 
-            }        
+            }
+            if (ingestDocument.hasField("container_image")) {
```

Comment:
should it be `else if` here? I am considering the case that document has both `kubernetes` and `container_image` field.

Reply:
I think container_image would supercede it in that case so I think this logic is OK

Reply:
cool.